src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -336,10 +338,10 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
                 BuiltinFunctionsCore.Date,
                 StandardErrorHandling<NumberValue>(
                     expandArguments: NoArgExpansion,
-                    replaceBlankValues: DoNotReplaceBlank,
+                    replaceBlankValues: ReplaceBlankWithZero,
                     checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
                     checkRuntimeValues: ExactSequence(
-                        PositiveNumericNumberChecker,
+                        FiniteChecker,
```

Comment:
What additional scenario(s) is this accounting for?

Reply:
The validation is already done in the implementation. And the PositiveNumericNumberChecker returns a Error(Kind=Numeric), which should be used for numeric functions only; the actual error type should be InvalidArgument